code/ui/manager/src/app.tsx

```diff
@@ -27,9 +24,10 @@ export interface AppProps {
   viewMode: State['viewMode'];
   layout: State['layout'];
   panelCount: number;
+  mains: Page[];
```

Comment:
I was indecicsive on what to call this feature..

"main", "pages", "views"..
Nothing felt quite right.

I ended up using pages, mostly because `settings` was defined as a page.